app/views/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.register.container
+  .row
+    .col-sm-8.col-sm-offset-1
+      = bootstrap_form_for @user, layout: :horizontal do |f|
+        %header
+          %h1 Register
+        %fieldset
+          = f.text_field :email_address, label: 'Email Address', class:'form-control'
```

Comment:
You shouldn't need the `form-control` class. `bootstrap_form_for` should take care of that for you.